### PR TITLE
refactor cmd

### PR DESCRIPTION
**Pull request by uptutu**
I do like what this #47 described, refactor the `/cmd`.

### CODE REVIEW

File: pkg/config/config.go [hunk 1]
```diff
@@ -95,17 +96,16 @@ func LoadStandaloneConfiguration(configPath string) (*Configuration, error) {
...
-	boolVar(&c.Log.Dev, "debug", getEnvBool("TMANAGER_DEBUG", false), "enable debug mod.")
-	strVar(&c.Log.Level, "log.level", getEnvStr("TMANAGER_LOG_LEVEL", "debug"), "log level(default debug).")
-	strVar(&c.HTTPAddr, "http.addr", getEnvStr("TMANAGER_HTTP_ADDR", ":31234"), "http listen address(default :31234).")
-	strVar(&c.GRPCAddr, "grpc.addr", getEnvStr("TMANAGER_GRPC_ADDR", ":31233"), "grpc listen address(default :31233).")
-	strVar(&c.Dapr.GRPCPort, "dapr.port", getEnvStr("DAPR_GRPC_PORT", "50001"), "dapr grpc listen address(default 50001).")
-	strVar(&c.Dapr.PrivateStateName, "dapr.private_state_name", getEnvStr("DAPR_PRIVATE_STATE_NAME", "keel-private-store"), "dapr private store name(default keel-private-store).")
-	strVar(&c.Dapr.PublicStateName, "dapr.public_state_name", getEnvStr("DAPR_PUBLIC_STATE_NAME", "keel-public-store"), "dapr public store name(default keel-public-store).")
-	strVar(&c.Tkeel.Secret, "tkeel.secret", getEnvStr("TKEEL_SECRET", "changeme"), "tkeel secret.(default changeme)")
-	strVar(&c.Tkeel.Version, "tkeel.version", getEnvStr("TKEEL_VERSION", "v0.2.0"), "tkeel version.(default v0.2.0)")
+func (c *Configuration) AddFlags(fs *pflag.FlagSet) {
```
Comment (OdysseusC) [inline pkg/config/config.go]: 这个函数名感觉 Attach 好一点。

File: cmd/rudder/rudder.go [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+	cmd, err := newRootCmd()
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "%+v", err)
+		os.Exit(1)
+	}
+
+	// run when each command's execute method is called.
+	cobra.OnInitialize(func() {
+		// you can do something here before the cmd run.
+	})
```
Comment (OdysseusC) [inline cmd/rudder/rudder.go:53]:
```suggestion

```
这几句去掉？

Comment (OdysseusC) [inline cmd/rudder/rudder.go]: 这几行去掉？不是 template。

File: cmd/rudder/rudder.go [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+	"github.com/tkeel-io/tkeel/pkg/model/proute"
+	"github.com/tkeel-io/tkeel/pkg/server"
+	"github.com/tkeel-io/tkeel/pkg/service"
+)
+
+var (
+	configFile string
+
+	longUsage = `rudder is the main control plugin in the tkeel platform.
+	Used to manage plugins and tenants.`
```
Comment (OdysseusC) [inline cmd/rudder/rudder.go]: const?

File: cmd/rudder/rudder.go (hunk 2 above)
Comment (uptutu) [reply cmd/rudder/rudder.go]: I removed this.

File: pkg/config/config.go (hunk 1 above)
Comment (uptutu) [reply pkg/config/config.go]: I changed this to AttachConfig.

**Pull request closed without merging**